packages/server/test/websockets.test.ts

```diff
@@ -64,18 +66,17 @@ function factory(config?: { createContext: () => Promise<any> }) {
       .subscription('onMessage', {
         input: z.string().nullish(),
         resolve() {
-          const sub = (subRef.current = new trpc.Subscription<Message>(
-            (emit) => {
-              const onMessage = (data: Message) => {
-                emit.data(data);
-              };
-              ee.on('server:msg', onMessage);
-              return () => {
-                subscriptionEnded();
-                ee.off('server:msg', onMessage);
-              };
-            },
-          ));
+          const sub = observable<Message>((emit) => {
```

Comment:
Should we add some tests for `emit.error(...)` and `emit.complete()`?

Reply:
Good idea, will try to do that soon

Reply:
They are already tested (the variable is called `subRef.current`)